scripting/src/main/java/org/apache/unomi/scripting/MvelScriptExecutor.java

```diff
@@ -24,11 +24,20 @@
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
 
+/**
+ * MVEL script executor implementation
+ */
 public class MvelScriptExecutor implements ScriptExecutor {
 
+    private final static String INVALID_SCRIPT_MARKER = "Invalid String Marker";
```

Comment:
is it correct the placeholder text, or typo (script // string)?

Reply:
It's a typo I'll correct it.